src/request/sign-swap/SignSwapApi.js

```diff
@@ -95,6 +95,13 @@ class SignSwapApi extends PolygonRequestParserMixin(BitcoinRequestParserMixin(To
                 fee: this.parsePositiveInteger(request.fund.fee, true, 'fund.fee'),
                 bankLabel: this.parseLabel(request.fund.bankLabel, true, 'fund.bankLabel'),
             };
+        } else if (request.fund.type === 'CRC') {
+            parsedRequest.fund = {
+                type: 'CRC',
+                amount: this.parsePositiveInteger(request.fund.amount, false, 'fund.amount'),
+                fee: this.parsePositiveInteger(request.fund.fee, true, 'fund.fee'),
+                recipientLabel: this.parseLabel(request.fund.recipientLabel, true, 'fund.recipientLabel'),
```

Comment:
As this is the funding side, the label is either the `senderLabel`, or, like in EUR, a `sinpeLabel`.

Reply:
`sinpeLabel`